internal/notify/notifications.lua

```diff
@@ -303,6 +307,34 @@ local function get_bar(get_fn, get_max_fn, text, color)
     return nil
 end
 
+local function get_save_alert()
+    local mins_since_save = dfhack.persistent.getUnsavedSeconds()//60
+    local pen = COLOR_LIGHTCYAN
+    if mins_since_save < save_time_threshold_mins then return end
+    if mins_since_save >= 4*save_time_threshold_mins then
+        pen = COLOR_LIGHTRED
+    elseif mins_since_save >= 2*save_time_threshold_mins then
+        pen = COLOR_YELLOW
+    end
+    return {
+        {text='Last save: ', pen=COLOR_WHITE},
+        {text=dfhack.formatInt(mins_since_save) ..' mins ago', pen=pen},
+    }
+end
+
+local function save_popup()
+    if dfhack.world.isFortressMode() then
+        local mins_since_save = dfhack.persistent.getUnsavedSeconds()//60
+        local message = 'It has been ' .. dfhack.formatInt(mins_since_save) .. ' minutes since your last save. \n\nWould you like to save now? ' ..
+        '(Note: You can also close this reminder and save manually)'
+        dlg.showYesNoPrompt('Save now?', message, nil, function () dfhack.run_script('quicksave') end)
+    else
+        local adv_warn_msg = "Unfortunately, quicksave doesn't currently work in Adventure mode, so you will need to save manually.\n\n"..
+        'Close this popup to open the options menu and select "Save and continue playing"'
+        dlg.showMessage("Unable to use quicksave", adv_warn_msg, COLOR_WHITE, function () gui.simulateInput(dfhack.gui.getDFViewscreen(true), 'OPTIONS') end)
+    end
```

Comment:
```suggestion
    local mins_since_save = dfhack.persistent.getUnsavedSeconds()//60
    local message = 'It has been ' .. dfhack.formatInt(mins_since_save) .. ' minutes since your last save.'
    if dfhack.world.isFortressMode() then
        message = message .. '\n\nWould you like to save now? (Note: You can also close this reminder and save manually)'
        dlg.showYesNoPrompt('Save now?', message, nil, function() dfhack.run_script('quicksave') end)
    else
        message = message .. '\n\nClose this popup to open the options menu and select "Save and continue playing"'
        dlg.showMessage('Save reminder', message, COLOR_WHITE, function() gui.simulateInput(dfhack.gui.getDFViewscreen(true), 'OPTIONS') end)
    end
```